app/src/main/java/com/example/composetemplate/utils/DateUtil.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.composetemplate.utils
+
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
+
+class DateUtil {
+
+    companion object {
+        const val SERVER_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"  //"2021-04-26T10:00:00Z"
```

Comment:
I think I asked that before in other PR, there is special reason for it to be companion object? I think it will be better to make it an extension function of Long

Reply:
todo: Ask guy